### PR TITLE
New readme

### PR DESCRIPTION
**Pull request by swapdisk**
Here's my first draft for the new main README. I still need to do the README files for each of the individual roles, but figured I'd get the high-level discussion going with this one.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,83 +1,32 @@
...
-<!-- Galaxy will eventually list the module docs within the UI, but until that is ready, you may need to either describe your plugins etc here, or point to an external docsite to cover that information. -->
-
-## Using this collection
-
-<!--Include some quick examples that cover the most common use cases for your collection content. It can include the following examples of installation and upgrade (change infra.leapp correspondingly):-->
-
-### Installing the Collection from Ansible Galaxy
+## Installing the collection from Ansible Galaxy
 
 Before using this collection, you need to install it with the Ansible Galaxy command-line tool:
```
Comment (swapdisk) [inline README.md:31]: @heatmiser @djdanielsson @jeffmcutter:
Question for you Ansible gurus. This whole "Installing the collection from Ansible Galaxy" section is copied from the original README boilerplate. Just curious if these commands are relevant or if we just need to give the requirements.yml example assuming people will be using the collection from AAP?

Comment (swapdisk) [reply README.md:31]: As discussed in the meeting, we'll leave this as is.

**Comment by swapdisk** [comment]
I've added the README files for the roles that we'll be keeping in the collection. This is ready to review and merge if it looks good.

**Review by djdanielsson — APPROVED** [review]
LGTM

**Pull request merged**